src/views/studio/MyVideosView/MyVideos.styles.ts

```diff
@@ -28,6 +29,10 @@ export const TabsContainer = styled.div`
   }
 `
 
+export const StyledGrid = styled(Grid)`
+  grid-template-rows: 1fr 1fr;
```

Comment:
What's the point of this?

Reply:
This is needed for mobile views.
without: 
![image](https://user-images.githubusercontent.com/51168865/134198749-22382605-a45c-4fc4-a6bc-92027e93fa3f.png)
and with this property: 
![image](https://user-images.githubusercontent.com/51168865/134198898-f008041e-fe4a-4860-8543-d3ffc8994c45.png)

I guess it's a workaround, I couldn't figure out which property I need set here to not shrink this tile.

Reply:
All right. I think this will work better:
```suggestion
  grid-auto-rows: 1fr;
```